test/reporters/base.spec.js

```diff
@@ -164,6 +164,34 @@ describe('Base reporter', function() {
         '      \n      actual expected\n      \n      a foobar inline diff\n      '
       );
     });
+
+    it("should truncate overly long 'actual' ", function() {
+      var actual = '';
```

Comment:
Hello, I'm trying to contribute to Mocha and before I do I'm trying to get idea of mocha's coding style. I want to know  why you used 'var' instea of 'const' or 'let'.  Thank you in advance.